src/main/java/com/ibm/devops/connect/CRPipeline/UploadDeployment.java

```diff
@@ -193,12 +193,12 @@ public void perform(final Run<?, ?> build, FilePath workspace, Launcher launcher
             }
         }
         if (succeeded != null && !succeeded.equals("")) {
-            payload.put("result", succeeded.equals("true") ? "Success" : "Failed");
+            payload.put("result", succeeded.equals("true") ? "success" : "failed");
```

Comment:
these should be changed back. The API expects the strings to be in a specific format (different between builds and deployments, it sucks I know)